app/utils/captcha.js

```diff
@@ -1,15 +1,69 @@
 const request = require('request')
 const logger = require('./logger')(__filename)
 
-const { GOOGLE_RECAPTCHA_SECRET_KEY, GOOGLE_RECAPTCHA_VERIFY_URL } = process.env
+const {
+  GOOGLE_RECAPTCHA_SECRET_KEY,
+  GOOGLE_RECAPTCHA_VERIFY_URL,
+  GOOGLE_RECAPTCHA_SITE_KEY,
+  GOOGLE_RECAPTCHA_USE_ENTERPRISE_VERSION
+} = process.env
 const HTTP_SUCCESS_CODE = 200
 
-function verifyCAPTCHAToken(token) {
+function verifyCAPTCHAEnterpriseVersion(token) {
   return new Promise((resolve, reject) => {
     if (!GOOGLE_RECAPTCHA_SECRET_KEY) {
       reject(new Error('reCAPTCHA secret key not set in environment'))
       return
     }
+    if(!token) {
+      reject(new Error('no reCAPTCHA enterprise widget token response provided'))
+      return
+    }
+    request.post({
+      url: GOOGLE_RECAPTCHA_VERIFY_URL,
+      proxy: process.env.http_proxy,
+      qs: {
+        key: GOOGLE_RECAPTCHA_SECRET_KEY
+      },
+      body: {
+        event: {
+          token: token,
+          siteKey: GOOGLE_RECAPTCHA_SITE_KEY
+        }
+      },
+      json: true
+    }, (error, response, body) => {
+      if (error) {
+        reject(error)
+        return
+      }
+      if (response.statusCode === HTTP_SUCCESS_CODE) {
+        // https://cloud.google.com/recaptcha-enterprise/docs/interpret-assessment
+        if (!body.score || body.score < 0.9) {
```

Comment:
Do we think it's worth having this score as environment variable so we can more easily tinker with it? Keeping it as hardcoded and having to deploy to change is probably fine though.
Would a comment about why we reject below a score of 0.9 be useful regardless?